content/en/docs/Guides/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Guides
```

Comment:
I would suggest not to use a capital 'Guides' in the directory path. Everything else is lowercase otherwise.